production/loki-mixin/dashboards/dashboard-loki-operational.json

```diff
@@ -2843,7 +2843,7 @@
           },
           "targets": [
             {
-              "expr": "{cluster=\"$cluster\", namespace=\"$namespace\", job=\"$namespace/ingester\"} |= \"level=error\"",
+              "expr": "{cluster=\"$cluster\", namespace=\"$namespace\", job=\"$namespace/ingester\"} | logfmt | level=\"error\"",
```

Comment:
Curious why need this change? (converting line filter -> logfmt + label filter)?

Is it for consistency sake? (asking because people often prefer simple line filter in cases like this, which also bit performant over label filter).

Reply:
I think for this particular set of queries it's not too bad, but for more complex queries logfmt plus label selectors reads nicer imo. Swapping operational dashboard queries to use logfmt was just on the todo list of our dashboard cleanup issue.